conversation_visitor_identity.go

```diff
@@ -0,0 +1,34 @@
+package hubspot
+
+const (
+	visitorIdentificationBasePath = "/conversations/v3/visitor-identification"
+)
+
+type IdentificationTokenResponse struct {
+	Token string `json:"token"`
+}
+
+type IdentificationTokenRequest struct {
+	FirstName string `json:"firstname"`
```

Comment:
In the document, it is `firstName`, but it worked with `firstname` at my environment, so there seems to be no problem with the json tag as it is.
https://developers.hubspot.com/docs/api/conversation/visitor-identification

Reply:
Updated as requested!